README.md

```diff
@@ -129,6 +129,43 @@ result = whisper.decode(model, mel, options)
 print(result.text)
 ```
 
+## Use in 🤗 Transformers
+
+Whisper is available in the Hugging Face [Transformers](https://github.com/huggingface/transformers) library from Version 4.23.1, with both PyTorch and TensorFlow implementations. All the official checkpoints can be found on the [Hugging Face Hub](https://huggingface.co/models?search=openai/whisper), alongside [documentation](https://huggingface.co/docs/transformers/model_doc/whisper) and [examples scripts](https://huggingface.co/blog/fine-tune-whisper).
```

Comment:
Maybe might be good to mention how to install transformers:
```suggestion
Whisper is available in the Hugging Face [Transformers](https://github.com/huggingface/transformers) library from Version 4.23.1, with both PyTorch and TensorFlow implementations. All the official checkpoints can be found on the [Hugging Face Hub](https://huggingface.co/models?search=openai/whisper), alongside [documentation](https://huggingface.co/docs/transformers/model_doc/whisper) and [examples scripts](https://huggingface.co/blog/fine-tune-whisper).

To install transformers and datasets, make sure to run:
```bash
pip install transformers==4.24 datasets==2.6.1 librosa
```
```